scripts/packages/template_bin.sh

```diff
@@ -158,7 +164,7 @@ echo -n .
 ln -s "${base}/bin/bazel" "${bin}/bazel"
 echo -n .
 
-if [ "${UID}" -ne 0 ]; then
+if [ "$should_uncompress" = true ] && [ "${UID}" -ne 0 ]; then
```

Comment:
Wow. I didn't realize you can do logical AND with single brackets. Today I Learned!